### PR TITLE
Update tracker

### PR DESCRIPTION
**Pull request by ryrjanes**
I updated the read me to add the list of different trackers. I also updated the two gov trackers .js to fall inline with how the sites add them to the pages. This will reduce errors.

### CODE REVIEW

**Comment by danpollock** [comment]
One more tweak: We should say that people shouldn't add this without confirming with us first.

**Comment by ryrjanes** [comment]
I have made the change to the read me file regarding the to run section.

**Comment by ryrjanes** [comment]
Among the other changes, now I have updated to the new js that Snowplow provided. I have tested and all is working fine.

File: examples/gov.bc.ca_js_tracker/README.md [hunk 1]
```diff
@@ -2,13 +2,19 @@
 
 This code demonstrates the inline Javascript to track web analytics events for CMS Lite sites. The [`gov_sp_script.js`](./gov_sp_script.js) includes adding a custom context to include the Node Id. The [`gov_search_sp_script.js`](./gov_search_sp_script.js) search tracker includes a search event that will collect the search terms for gov. It only connects to Snowplow Mini and will need to be updated to the push data to the production Snowplow pipeline when development is complete.
 
+## Trackers
```
Comment (danpollock) [inline examples/gov.bc.ca_js_tracker/README.md]: Let's turn this into a bullet list.

**Pull request merged**